docs/styleguide/PyGuide.md

```diff
@@ -582,7 +582,8 @@ def load_state(model: torch.nn.Module, state_dict_to_load: dict, is_resume: bool
 
 Classes should have a docstring below the class definition describing the class. If your class
 has public attributes, they should be documented here follow the same formatting as a function's
-params section.
+params section. The `__init__` function and other magic methods may be left without a textual description,
+if there is nothing special about this exact implementation of the magic method.
```

Comment:
Could you specify what does `nothing special` mean? Maybe some samples when you should and should not add a docstring will clarify it.

Reply:
Added a few lines to clarify.